src/transdialog.cpp

```diff
@@ -702,32 +702,32 @@ bool mmTransDialog::ValidateData()
     //Checking account does not exceed limits
     if (m_new_trx || m_duplicate)
     {
-        bool abort_transaction = false;
-        double new_value = m_trx_data.TRANSAMOUNT;
-
-        if (m_trx_data.TRANSCODE == Model_Checking::all_type()[Model_Checking::WITHDRAWAL])
+        if ((m_trx_data.TRANSCODE == Model_Checking::all_type()[Model_Checking::WITHDRAWAL]) ||
+            (m_trx_data.TRANSCODE == Model_Checking::all_type()[Model_Checking::TRANSFER]))
         {
-            new_value *= -1;
-        }
+            const auto fromAccount{ Model_Account::instance().get(m_trx_data.ACCOUNTID) };
```

Comment:
We don't need this as we already have the account pointer in `account`.

Reply:
I agree. I will just use `account`.